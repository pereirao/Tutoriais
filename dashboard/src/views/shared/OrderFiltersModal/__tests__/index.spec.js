import React from 'react';
import moment from 'moment';
import OrderFilters from '../index';

it('renders correctly', () => {
  const all_filters = {
    start_date: moment('2017-01-01'),
    end_date: moment('2017-01-02'),
    types: ['delivery', 'shipping', 'pickup'],
    attributes: ['vip', 'gift', 'scheduled', 'exception']
  };
  expect(
    shallow(<OrderFilters initial_filters={all_filters} />)
  ).toMatchSnapshot();
});
