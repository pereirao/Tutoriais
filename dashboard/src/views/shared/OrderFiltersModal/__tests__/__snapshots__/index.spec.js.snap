// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="wrapper"
>
  <MBButton
    button_type="secondary"
    size="large"
  >
    <MBIcon
      color="mb_black"
      icon="filter"
    />
    <p
      className="filterButtonText"
    >
      Filter
    </p>
  </MBButton>
</div>
`;
